server/src/test/java/io/crate/integrationtests/InsertIntoIntegrationTest.java

```diff
@@ -655,27 +652,18 @@ public void testInsertIntoGeoPointArray() throws Exception {
             new Object[]{3, new Double[][]{new Double[]{9.9, 10.10}, new Double[]{11.11, 12.12}}});
         execute("refresh table t");
         execute("select points from t order by id");
-        assertThat(response.rowCount(), is(3L));
-        assertThat(
-            (List<Object>) response.rows()[0][0],
-            contains(
-                is(new PointImpl(1.1, 2.2, JtsSpatialContext.GEO)),
-                is(new PointImpl(3.3, 4.4, JtsSpatialContext.GEO))
-            )
+        assertThat(response).hasRowCount(3L);
+        assertThat((List<Object>) response.rows()[0][0]).contains(
```

Comment:
here and below: `containsExactly`, that's one of the method naming I don't like so much in assertj :(